container_pipeline/pipeline_api/urls.py

```diff
@@ -0,0 +1,7 @@
+from container_pipeline.pipeline_api.v1.urls import get_v1_urls
+
+
+def get_api_urls():
+    urls = []
+    urls += get_v1_urls()
```

Comment:
return type of `get_v1_urls()` is already list, you can just

```
def get_api_urls():
    return get_v1_urls()
```

Reply:
I guess this would be good for now, so il get it done, but in future , when we have more versions, this will be the route we want to go